src/dirac_cwl_proto/execution_hooks/core.py

```diff
@@ -162,16 +264,49 @@ def pre_process(
         """
         return command
 
-    def post_process(self, job_path: Path, **kwargs: Any) -> bool:
+    def post_process(
+        self, job_path: Path, stdout: Optional[str] = None, **kwargs: Any
+    ) -> bool:
         """Post-process job outputs.
 
         Parameters
         ----------
         job_path : Path
             Path to the job working directory.
         """
+        # Get the outputs and outputted files from the cwltool standard output
+        if stdout:
+            outputs = self.get_job_outputted_files(stdout)
+            for output, file_paths in outputs.items():
+                self.store_output(output, file_paths)
         return True
 
+    def get_job_outputted_files(self, stdout: str) -> dict[str, list[str]]:
+        """Get the outputted filepaths per output.
+
+        Parameters
+        ----------
+        stdout : str
+            The console output of the the job
+
+        Returns
+        ----------
+        dict[str, list[str]]
+            The dict of the list of filepaths for each output
+        """
+        outputted_files: dict[str, list[str]] = {}
+        outputs = json.loads(stdout)
+        for output, files in outputs.items():
+            if files:
+                if not isinstance(files, List):
+                    files = [files]
+                file_paths = []
+                for file in files:
+                    if file:
+                        file_paths.append(str(file["path"]))
+                outputted_files[output] = file_paths
```

Comment:
Very minor and a bit more pythonic: 
```suggestion
        outputted_files: dict[str, list[str]] = {}
        outputs = json.loads(stdout)
        
        for output, files in outputs.items():
            if not files:
                continue
                
            if not isinstance(files, List):
                files = [files]
            file_paths = []
            for file in files:
                file_paths.append(str(file["path"]))
            outputted_files[output] = file_paths
```